src/bin/nydus-image/main.rs

```diff
@@ -1038,6 +1091,53 @@ impl Command {
         OutputSerializer::dump(matches, build_output, build_info)
     }
 
+    fn chunkdict_save(matches: &ArgMatches) -> Result<()> {
+        let bootstrap_path = Self::get_bootstrap(matches)?;
+        let config = Self::get_configuration(matches)?;
+        let db_path = matches.get_one::<String>("database-path").unwrap();
+        debug!("db_path: {}", db_path);
+        // For backward compatibility with v2.1.
+        config
+            .internal
+            .set_blob_accessible(matches.get_one::<String>("bootstrap").is_none());
+
+        let db_type: Vec<&str> = db_path.split("://").collect();
+
+        if db_type.is_empty() {
+            return Err(anyhow!("Invalid or unsupported database path. Please provide a correct database path, such as sqlite:///path/to/database.db"));
+        }
+
+        let blobs: Vec<Arc<nydus_storage::device::BlobInfo>> = match db_type[0] {
+            "sqlite" => {
+                let mut deduplicate: Deduplicate<SqliteDatabase> =
+                    Deduplicate::<SqliteDatabase>::new(bootstrap_path, config, db_type[1])?;
+                deduplicate
+                    .save_metadata(Some(&db_path))
+                    .with_context(|| format!("failed to check bootstrap {:?}.", bootstrap_path))?
+            }
+            _ => {
+                return Err(anyhow!("Unsupported database type: {}. Please provide a correct database type, such as sqlite.", db_type[0]));
```

Comment:
`bail!("Unsupported database type: {}, please use a valid database URI, such as 'sqlite:///path/to/database.db'.")`